### DISCUSSION TITLE
Feature Request: Multiple audio track support

### DISCUSSION THREAD

**Discussion by CUR5EDFLVME** [huynhsontung/Screenbox#394] [Ideas]
I haven't really seen video players that support multiple audio tracks at the same time (maybe VLC, but you can choose between different audio tracks), it's extremely handy when you record stuff with multiple audio tracks(microphone, music player, discord, videogame) and you need to somehow watch it. So the problem is that mp4 can contain all this audio information and you can't for some reason hear all this stuff, except only one audio track. And the only way to hear all the stuff you need is using premiere pro or any other pretty much heavy video editing software, there's no convenient way to do this.
I don't know is it even possible to add this feature, but man, if you make it, it would be literally the best videoplayer at least for people who use it to watch recordings with multiple audio tracks. Thank you for your work, keep it up!!!

**Comment by KingDigweed** [comment]
@huynhsontung I would also really appreciate such a feature. I've been used to achieving this with VLC by using the registry to add the following as a custom shell command. When I right click any .mp4 file in Windows Explorer, I then get an option to play the video in VLC with all the audio tracks playing too:
`"C:\Program Files\VideoLAN\VLC\vlc.exe" --sout-all --sout #display "%1%"`

It works for my use case, and would probably for @CUR5EDFLVME too as it just plays all available audio tracks at once. I appreciate this may not be such a popular feature request, but there are some of us out there! I would be happy to solve this just like I did with VLC using a command, but I've not been able to find the application path for use in the command above because Screenbox was installed using the Microsoft Store and that complicates things in this regard.

If you have a workaround you can suggest for me to try, or are able to implement such a feature, it would be very much appreciated!
